storage/object/how-to/get-started-edge-services.mdx

```diff
@@ -0,0 +1,257 @@
+
+9. Click **Customize domain** to finish.
+
+Your customized domain is set up, and you are returned to the Edge Services dashboard. The customized domain displays in the Endpoint panel. When you access your bucket through this domain, it's content will be served via Edge Services.
+
+### How to edit your customized domain or its certificate
+
+After customizing your domain, you can edit it (or its certficate) at any time as follows:
+
+1. Click **Object Storage** in the **Storage** section of the side menu. The list of your buckets displays.
+
+2. Click the name of the bucket you want to configure a custom domain for.
+
+3. Click the **Edge Services** tab.
+
+    TODO SCREENSHOT
+
+4. In the **Endpoint** panel, click **Edit**. The following screen displays:
+
+    TODO SCREENSHOT
+
+5. Edit the subdomain as desired - do not forget to also set up a new CNAME record if necessary. 
+
+6. Select a different certificate from Secret Manager, or manually import a new certificate for your custom domain.
+
+7. Click **Edit domain** to finish.
+
+### How to reset your customized domain to the original Edge Services endpoint
+
+Even though the original Edge Services endpoint (e.g. `https://bucket-name.edge.scw.cloud`) will continue to work after you add a customized domain, you can choose to remove your customized domain completely and go back to the original Edge Services endpoint only. This is done via the reset function:
+
+1. Click **Object Storage** in the **Storage** section of the side menu. The list of your buckets displays.
+
+2. Click the name of the bucket you want to configure a custom domain for.
+
+3. Click the **Edge Services** tab.
+
+    TODO SCREENSHOT
+
+4. In the **Endpoint** panel, click **Reset**. 
+
+    A screen displays warning you that this will reset the bucket's domain back to the default Edge Services endpoint. Edge Services will consider your customized subdomain as unknown. You should also remember to delete your CNAME record from your domain provider, and your SSL/TLS certificate.
+
+5. Click **Reset domain** to finish.
+
+## How to configure your cache
+
+The cache feature allows you to cache your bucket's content with Edge Services. This means that content can be served directly to users from Edge Services' servers, enhancing performance. You can disable and enable caching at will, as well as control the lifetime of an object in the cache, and purge your cache at any time.
+
+### How to enable your cache
+
+1. Click **Object Storage** in the **Storage** section of the side menu. The list of your buckets displays.
+
+2. Click the name of the bucket you want to configure a cache for.
+
+3. Click the **Edge Services** tab.
+
+    TODO SCREENSHOT
+
+4. In the **Cache** panel, use the <Icon name="toggle" /> icon to enable the cache.
+
+    The **Lifetime** configuration box displays. This enables you to define, in minutes, how long an object can be stored in the cache before it must be retrieved freshly from the bucket.
```

Comment:
@RoRoJ Le lifetime is in seconds

Reply:
but may change (cf https://scaleway.slack.com/archives/C0446VC22MV/p1695888657852059)

Reply:
Thanks, corrected